docs/src/main/sphinx/release/release-464.md

```diff
@@ -0,0 +1,41 @@
+# Release 464 (30 Oct 2024)
+
+## General
+
+* {{breaking}} Require JDK 23 to run Trino. ({issue}`21316`)
+* Add the [](/connector/faker) to allow easy generation of data. ({issue}`23691`)
+* Add the Vertica connector. ({issue}`23948`)
+* Rename the
+  `fault-tolerant-execution-eager-speculative-tasks-node_memory-overcommit`
+  configuration property to
+  `fault-tolerant-execution-eager-speculative-tasks-node-memory-overcommit`.
+  ({issue}`23876`)  
+
+## Accumulo connector
+
+* {{breaking}} Remove the Accumulo connector. ({issue}`23792`)  
+
+## Delta Lake connector
+
+* Fix S3 file listing of buckets that enforce requester pays. ({issue}`23906`)
```

Comment:
Is the listing inaccurate? Or causing errors? Maybe also worth linking to https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html on the "requester pays" text.

Reply:
Links sounds good .. wdyt @martint and @pettyjamesm ?

Reply:
Query used to just fail .. so "Fix" is accurate .. unless we want to say something "Prevent queries from failing when ... "

Reply:
I'm good with linking to the docs, and yeah- the listing would fail before (so not a correctness issue).

Reply:
I'd say "Fix failure" then, just to make it clear what's being fixed.

Reply:
Done